easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'Bundle'
+
+name = 'deepTools'
+version = '3.3.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://deeptools.readthedocs.org/'
+description = """deepTools is a suite of python tools particularly developed for the efficient analysis of
+ high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('Pysam', '0.15.1', versionsuffix),
+    ('matplotlib', '3.0.0', versionsuffix),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
+
+exts_list = [
+    ('py2bit', '0.3.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/py2bit'],
+        'checksums': ['450555c40cba66957ac8c9a4b6afb625fb34c4bb41638de78c87661ff8b682ef'],
+    }),
+    ('pyBigWig', '0.3.17', {
+        'modulename': 'pyBigWig',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyBigWig'],
+        'checksums': ['41f64f802689ed72e15296a21a4b7abd3904780b2e4f8146fd29098fc836fd94'],
+    }),
+    ('deeptoolsintervals', '0.1.9', {
+        'source_urls': ['https://pypi.python.org/packages/source/d/deeptoolsintervals'],
+        'checksums': ['7d94c36fd2b6f10d8b99e536d2672e8228971f1fc810497d33527bba2c40d4f6'],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/d/deepTools'],
+        'checksums': ['514240f97e58bcfbf8c8b69ae9071d26569b491f089e1c1c46ba4866d335e322'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/bamCompare', 'bin/bamCoverage', 'bin/bamPEFragmentSize', 'bin/computeGCBias', 'bin/computeMatrix',
+              'bin/correctGCBias', 'bin/multiBamSummary', 'bin/plotCorrelation', 'bin/plotCoverage',
+              'bin/plotHeatmap', 'bin/plotProfile'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = ["python -c 'import deeptools'"]
```

Comment:
The `import` test is already done for the `deepTools` extension (but it doesn't hurt to check again, I guess?)

Reply:
Also done.